docs/source/highlights.md

```diff
@@ -64,11 +65,13 @@ affine = Affine(
 # convert the image using bilinear interpolation
 new_img = affine(image, spatial_size=(300, 400), mode='bilinear')
 ```
+Experiments and test results are available at [Fused transforms test](https://github.com/Project-MONAI/MONAI/blob/master/examples/notebooks/transform_speed.ipynb).
+
+Currently all the geometric image transforms (Spacing, Zoom, Rotate, Resize, etc.) are designed based on the PyTorch native interfaces (instead of using scipy, scikit-image, etc.). [Geometric transforms tutorial](https://github.com/Project-MONAI/MONAI/blob/master/examples/notebooks/3d_image_transforms.ipynb) indicates the usage of affine transforms with 3D medical images.
```

Comment:
could remove " (instead of using scipy, scikit-image, etc.)"

Reply:
removed in the latest commit.
Thanks.